index.js

```diff
@@ -130,7 +130,10 @@ function asPromise(opts) {
 			}
 
 			if (err) {
-				err.response = response;
+				Object.defineProperty(err, 'response', {
+					value: response,
+					enumerable: false
```

Comment:
Didn't know that, but I feel, that this way it is clearer why this code is here, compared with `Object.defineProperty(err, 'response', {value: response});` - someone definatly will PR `err.response = response;` :smile: